### DISCUSSION TITLE
Trilium is blocked

### DISCUSSION THREAD

**Discussion by enzofoucaud** [TriliumNext/Trilium#1471] [Q&A]
Hello thank to @zadam for this app,

But I have a problem since a bluescreen come...
I reinstall the app and delete all file and I came back to the starting point still blocked

Screen : https://prnt.sc/vue5tr

**Comment by zadam** [comment]
Hi, so Trilium used to work, then your computer crashed with BSOD and now it doesn't work anymore?

I remember seeing something like this and it was related to graphics driver ...

**Comment by enzofoucaud** [comment]
I just reinstalled the graphics driver and nothing changes

**Reply by enzofoucaud** [reply]
I have shutdown my computer and now it's ok. I don't know why i had this issue ... but now it's all good.
